api/cpp/include/slint.h

```diff
@@ -984,11 +984,7 @@ struct ReverseModelInner : private_api::ModelChangeListener
 
     void row_removed(size_t first_removed_row, size_t count) override
     {
-        auto row_count = source_model->row_count();
-        auto old_row_count = row_count + count;
-        auto row = old_row_count - first_removed_row - 1;
-
-        target_model.row_removed(row, count);
+        target_model.row_removed(source_model->row_count() - first_removed_row, count);
```

Comment:
I've not made a test for this because there is currently no builtin model in C++ that can remove several row in one operation, and i was too lazy to make one.  (but note that the previous test are still passing, this equivalent to the old code when `count` is 1 because the old code was equivalent `row_count + count - first_removed_row - 1`)